pkg/iso/http.go

```diff
@@ -0,0 +1,75 @@
+package iso
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/kubev2v/migration-planner/internal/util"
+)
+
+const (
+	rhcosUrl string = "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/rhcos-live.x86_64.iso"
+)
+
+var (
+	theoreticalTotalSize int64 = 1*1024*1024*1024 + 300*1024*1024 // ~ 1.3 Gb
```

Comment:
Why do we need this?

Reply:
not really. I put it just in case the Content-Length is not present.  I'll remove it.

Reply:
Yeah, let's ignore the check if content-lenght header is not present